src/OpenConext/Profile/Value/Consent.php

```diff
@@ -95,4 +113,12 @@ public function getConsentType()
     {
         return $this->consentType;
     }
+
+    /**
+     * @return null|AttributeSet
+     */
+    public function getAttributes()
+    {
+        return $this->attributes;
```

Comment:
oh, it was discussed, just evolving design and me making a general remark :wink: